images/fediverse.svg

```diff
@@ -123,11 +121,12 @@
      units="px"
      inkscape:pagecheckerboard="0"
      inkscape:deskcolor="#d1d1d1"
-     inkscape:window-width="1193"
-     inkscape:window-height="1478"
-     inkscape:window-x="0"
-     inkscape:window-y="24"
-     inkscape:window-maximized="1">
+     inkscape:window-width="1421"
+     inkscape:window-height="793"
+     inkscape:window-x="269"
+     inkscape:window-y="178"
+     inkscape:window-maximized="1"
+     inkscape:document-rotation="0">
```

Comment:
I really dislike the amount of change in this SVG. What's weird is that your Inkscape is older, so it's likely to get changed *back* if someone else touches this SVG. My thought is that the only lines we actually needed to change here are the changes *below this comment* where the fill color is changed. Can we change only the seven lines that were changed below, and not change any of the document window or metadata stuff?